src/components/ContractEventDataPanel.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <app-panel class="contract-event-data-panel">
+    <div class="contract-event-data-panel__heading">
+      Arguments
+    </div>
+    <dl>
+      <template
+        v-for="(arg, index) in event.data"
```

Comment:
```suggestion
        v-for="(arguments, index) in event.data"
```

I would personally better read arguments than arg, if there are no technical limitations for shortening

Reply:
Right, fixed :+1: